refine.ini

```diff
@@ -8,10 +8,10 @@ no_proxy="localhost,127.0.0.1"
 
 # Memory and max form size allocations
 #REFINE_MAX_FORM_CONTENT_SIZE=1048576
-REFINE_MEMORY=1400M
+REFINE_MEMORY=3500M
```

Comment:
1400M is not enough any more?

Reply:
that's just a mistake - I did not intend to change this, it's just my own setting that got pushed with the rest, I will revert